spec/spec_helper.rb

```diff
@@ -121,6 +121,7 @@ module Shell
 
   config.filter_run_excluding :skip_appveyor => true if ENV["APPVEYOR"]
   config.filter_run_excluding :appveyor_only => true unless ENV["APPVEYOR"]
+  config.filter_run_excluding :skip_travis => true if ENV["TRAVIS"]
```

Comment:
Can you add a comment here explaining _why_ we need to skip certain tests on Travis?